c_src/filewatch_inotify.c

```diff
@@ -9,9 +13,12 @@
 #include <ei.h>
 #include <erl_driver.h>
 
-#include "macrology.h"
-
-enum { MAX_COOLDOWNS = 12, MSG_LENGTH = 11 };
+enum {
+    MAX_EVENTS = 12,
+    EVENT_TUPLE_LENGTH = 11,
+    PORT_CALL_ARG = 1,
+    WATCH_FLAGS = IN_MOVED_TO | IN_CLOSE_WRITE | IN_ATTRIB
```

Comment:
IIRC this is a remnant from me trying to debug gateway not updating when the staging bert files did not trigger filewatch when they were rsynced.